docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: '3.9'
```

Comment:
Why is there docker stuff in this branch?

Reply:
@SimonR99 insisted and all branches written during the competition have been merged in enrich.